frappe/public/js/onboarding.bundle.js

```diff
@@ -0,0 +1,344 @@
+frappe.ui.OnboardingTour = class OnboardingTour {
```

Comment:
Bundle files should ideally just import code from other files. This way code can be split and reused. 

Even if it's not required, let's move it to separate file for consistency as this is quite a huge file.